index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <title>My-Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div id="nav-container">
+                <div class="shapes">
+                    <img src="assets/shapes/image_geometry_1.jpg" alt="">   
+                </div>
```

Comment:
- [ ] Kindly make sure that  background image is used as a CSS background-image image and as in a `img` tag as required for the project and for best practices.
![Screenshot from 2023-02-16 02-20-09](https://user-images.githubusercontent.com/83042742/219226089-4c784b5f-28ee-46e6-b1e7-593acd40eb1d.png)